docs/fr/users/06_Fever_API.md

```diff
@@ -1,23 +1,135 @@
+
+* MacOS
+  * [ReadKit](https://apps.apple.com/app/readkit/id588726889) (Commercial)
+
+
+## Fonctionnalités
+
+Les fonctionnalités suivantes sont implémentées :
+
+* fetching categories
+* fetching feeds
+* fetching RSS items (new, favorites, unread, by_id, by_feed, by_category,
+  since)
+* fetching favicons
+* setting read marker for item(s)
+* setting starred marker for item(s)
+* setting read marker for feed
+* setting read marker for category
+* supports FreshRSS extensions, which use the `entry_before_display` hook
+
+Les fonctionnalités suivantes ne sont pas implémentées :
+
+* « Hot Links » car il n'y a encore rien dans FreshRSS qui soit similaire ou
+  qui puisse être utilisé pour le simuler.
+
+## Tester et déboguer
+
+If this API does not work as expected in your RSS reader, you can test it
+manually with a tool like [Postman](https://www.getpostman.com/).
+
+Configure a POST request to the URL
+https://freshrss.example.net/api/fever.php?api which should give you the
+result:
+```json
+{
+	"api_version": 3,
+	"auth": 0
+}
+```
+Great, the base setup seems to work!
+
+Now lets try an authenticated call. Fever uses an `api_key`, which is the
+MD5 hash of `"$username:$apiPassword"`.  Assuming the user is `kevin` and
+the password `freshrss`, here is a command-line example to compute the
+resulting `api_key`
+
+```sh
+api_key=`echo -n "kevin:freshrss" | md5sum | cut -d' ' -f1`
+```
+
+Add a body to your POST request encoded as `form-data` and one key named
+`api_key` with the value `your-password-hash`:
+
+```sh
+curl -s -F "api_key=$api_key" 'https://freshrss.exemple.net/api/fever.php?api'
+```
+
+This should give:
+```json
+{
+	"api_version": 3,
+	"auth": 1,
+	"last_refreshed_on_time": "1520013061"
+}
+```
+Perfect, you're now authenticated and you can start testing the more
+advanced features. To do so, change the URL and append the possible API
+actions to your request parameters. Please refer to the [original Fever
+documentation](https://feedafever.com/api) for more information.
+
+Some basic calls are:
+
+* https://freshrss.example.net/api/fever.php?api&items
+* https://freshrss.example.net/api/fever.php?api&feeds
+* https://freshrss.example.net/api/fever.php?api&groups
+* https://freshrss.example.net/api/fever.php?api&unread_item_ids
+* https://freshrss.example.net/api/fever.php?api&saved_item_ids
+* https://freshrss.example.net/api/fever.php?api&items&since_id=some_id
+* https://freshrss.example.net/api/fever.php?api&items&max_id=some_id
+* https://freshrss.example.net/api/fever.php?api&mark=item&as=read&id=some_id
+* https://freshrss.example.net/api/fever.php?api&mark=item&as=unread&id=some_id
+
+Remplacez `some_id` par un identifiant réel de votre base de données
+`freshrss_username_entry`.
+
+### Déboguer
+
+Si rien n'aide et que vos clients se conduisent toujours mal, ajoutez ces
+lignes à la début de `fever.api` :
```

Comment:
Si rien ne fonctionne correctement et que votre client se comporte étrangement, vous pouvez ajouter les quelques lignes suivantes au début du fichier `fever.api` pour déterminer la cause des problèmes rencontrés :

Reply:
This one's more than just a correction of my French; the original English is:

> If nothing helps and your clients still misbehaves, add these lines to the 
start of `fever.api`:

Setting aside the "your clients still misbehaves" that I overlooked previously, how about something like this?

If nothing helps and your client is still misbehaving, you can add the following lines to the beginning of the `fever.api` file to determine the cause of the problems:

Reply:
It does sound better